pages/faq.md

```diff
@@ -91,7 +91,7 @@ vagrant up
 You now have a working communityBeLL on your OS. 
 
 #### Q15: I named my repo incorrectly by not naming it &lt;username&gt;.github.io, can I rename it or I need to delete it?
-You need to delete it and start over. Navigate to the misnamed repo and click the settings tab or enter `https://github.com/YOUR-USERNAME/YOUR-MISNAMED-REPOSITORY/settings` in your browsers URL bar. Next, scroll down to the Danger Zone section of the settings and click '*Delete this repository*'. Read the warnings and then type in the name of the repo to confirm. 
+You need to delete it and start over. Navigate to the misnamed repo and click the settings tab or enter https://github.com/YOUR-USERNAME/YOUR-MISNAMED-REPOSITORY/settings in your browsers URL bar. Next, scroll down to the Danger Zone section of the settings and click '*Delete this repository*'. Read the warnings and then type in the name of the repo to confirm. 
```

Comment:
@sagun98 I got your idea to improve the user experience on mobile devices which is good. But when you change this line, the link becomes a hyperlink that directs to a 404 not found page which is not good. It would be great if you can work around this problem

Things I've reviewed:

[x] Check for issue number in pull request title
[x] Are there any unneeded files in the pull request?
[x] Did they make a branch for their patch?
[x] Does the pull request actually fix the issue?
[  ] Check the pull request on rawgit, does it display without any errors?
[x] Is there any merge conflicts?
[x] Make sure that people use their GitHub accounts when making commits through git